chatgse/app/locales/cn.ts

```diff
@@ -373,6 +373,22 @@ const cn = {
   FineTuned: {
     Sysmessage: "你是一个助手",
   },
+  About: {
```

Comment:
These we can keep for now, although chinese would probably be better at some point

Reply:
I translated them from English to simplified Chinese using Google Translate as a first version of these messages